iceoryx_hoofs/include/iceoryx_hoofs/internal/log/building_blocks/console_logger.inl

```diff
@@ -15,14 +15,14 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-#ifndef IOX_HOOFS_PLATFORM_BUILDING_BLOCKS_CONSOLE_LOGGER_INL
-#define IOX_HOOFS_PLATFORM_BUILDING_BLOCKS_CONSOLE_LOGGER_INL
+#ifndef IOX_HOOFS_LOG_BUILDING_BLOCKS_CONSOLE_LOGGER_INL
+#define IOX_HOOFS_LOG_BUILDING_BLOCKS_CONSOLE_LOGGER_INL
 
-#include "iceoryx_hoofs/log/platform_building_blocks/console_logger.hpp"
+#include "iceoryx_hoofs/log/building_blocks/console_logger.hpp"
 
 namespace iox
 {
-namespace pbb
```

Comment:
What did `pbb` stand for?

Reply:
platform building blocks ... usually I don't want to type that much :)